contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,45 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  mapping(uint80 => Strategy) internal _strategies;
+  mapping(string => uint80) internal _strategyNames;
+  mapping(bytes32 => IDCAStrategies.ShareOfToken) internal _tokenShares;
+  uint80 public strategyCounter; // NOTE: could we use _mintCounter from DCAStrategiesPermissionsHandler?
+
+  function _checkTokenSharesSanity(IDCAStrategies.ShareOfToken memory _tokens) internal pure returns (bool) {
+    // TODO
+    return true;
+  }
+
+  function _getTokenSharesKey(uint256 _strategyId, uint80 _version) internal pure returns (bytes32) {
```

Comment:
Since we might use it for other things, it would make sense to name it a little bit differently
```suggestion
  function _getStrategyAndVersionKey(uint256 _strategyId, uint80 _version) internal pure returns (bytes32) {
```